packages/framework/src/services/microservice/gateway/smb-gateway.service.test.ts

```diff
@@ -0,0 +1,77 @@
+import * as sinon from 'sinon';
+import { suite } from 'uvu';
+import * as assert from 'uvu/assert';
+
+import { CompleteTransportQueueItem } from '../../../models';
+import { SmbBrokerClient, SmbPublishPayload } from '../../smb/smb-broker.client';
+import { SmbGateway, SmbGatewayService } from './smb-gateway.service';
+
+const test = suite('SMB Microservice Gateway');
+
+test.before.each((context) => {
+  context.sandbox = sinon.createSandbox();
+});
+
+test.after.each((context) => {
+  context.sandbox.restore();
+});
```

Comment:
This can be removed since `context.sandbox` is reassigned in the before.each

Reply:
@alesmit without calling the `restore` the test begins to fail, probably the `createSandbox` is not enough and it has some caching logic under the hoods.